components/ui/colors/src/main/res/values/photon_colors.xml

```diff
@@ -12,119 +12,184 @@
         https://design.firefox.com/photon/visuals/color.html
```

Comment:
@brampitoyo The new color values are different from the ones in https://design.firefox.com/photon/visuals/color.html
Will the website be updated soon or should we change point future readers to another link?